test-caso0-cadena-auto.o: En la función `test_fecha(_fctkern_t*)':
/home/poo/POO/Roman_Aguilar_Rafael/P0/../Tests-auto/test-caso0-fecha-auto.cpp:31: definiciones múltiples de `test_fecha(_fctkern_t*)'
test-caso0-fecha-auto.o:/home/poo/POO/Roman_Aguilar_Rafael/P0/../Tests-auto/test-caso0-fecha-auto.cpp:31: primero se definió aquí
test-auto.o: En la función `test_fecha(_fctkern_t*)':
/home/poo/POO/Roman_Aguilar_Rafael/P0/../Tests-auto/test-caso0-fecha-auto.cpp:31: definiciones múltiples de `test_fecha(_fctkern_t*)'
test-caso0-fecha-auto.o:/home/poo/POO/Roman_Aguilar_Rafael/P0/../Tests-auto/test-caso0-fecha-auto.cpp:31: primero se definió aquí
/usr/lib/gcc/x86_64-linux-gnu/7/../../../x86_64-linux-gnu/Scrt1.o: En la función `_start':
(.text+0x20): referencia a `main' sin definir
collect2: error: ld returned 1 exit status
make: *** [test-P0-auto] Error 1
